src/components/nav-bar/nav-bar.css

```diff
@@ -1,31 +1,26 @@
 .navbar {
 	box-sizing: border-box;
-	padding: 0px 35px 0px 25px;
+	padding: 0 2.1875rem 0 1.5625rem;
```

Comment:
2.1875 is too long you can use 2.25rem (36px)
same for 1.5625, use 1.5rem (24px)